### ISSUE TITLE
Alert on loss of GPS lock

### ISSUE THREAD

**Issue by srob1** [centerclick/feedback#12]
It would be nice to be able to configure a notification from the device when it loses GPS lock -- possible alert mechanisms include syslog or e-mail or sms text message or snmp v1/v2 trap or v3 inform.

**Comment by dave4445** [comment]
Yes a SNMP trap or syslog would be good.  Note that GPS loss will currently cause a change of NTP stratum, NTP LED, and is visible in web interface and JSON data
